build/Dockerfile

```diff
@@ -9,8 +9,11 @@
 #   Red Hat, Inc. - initial API and implementation
 #
 
-FROM quay.io/bitnami/golang:1.14 as builder
-RUN go env GOPROXY
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
+FROM registry.access.redhat.com/ubi8-minimal:8.4-200.1622548483 as builder
```

Comment:
we should use https://catalog.redhat.com/software/containers/ubi8/go-toolset/5ce8713aac3db925c03774d1 (use the image instead of installing it)